c/tests/test_tables.c

```diff
@@ -723,6 +723,168 @@ test_edge_table(void)
     free(child);
 }
 
+static void
+test_edge_table_squash(void)
+{
+    int ret;
+    tsk_table_collection_t tables;
+    tsk_edge_table_t *edges;
```

Comment:
I don't think we need the edges pointer here, it's just confusing things I think.